guides/equality-diversity-and-inclusion/policy.md

```diff
@@ -0,0 +1,89 @@
+# Equality, Diversity & Inclusion Policy
+
+## About this policy
+
+This policy applies to everyone who works for Made Tech, or who acts on Made Tech’s behalf. All team members have a role in promoting equality, diversity and inclusion at work. We all have a personal responsibility to comply with the policy and to ensure, as far as possible, that others do the same.
+
+Made Tech is responsible for this policy, and for ensuring that all our staff understand their rights and obligations as detailed within it, and for any necessary training on equal opportunities.
+
+This policy does not form part of your contract of employment, and we may amend it at any time.
+
+## The legal framework
+
+It is illegal to discriminate against a person on the basis of the following [Protected Characteristics](https://www.gov.uk/discrimination-your-rights):
+
+- Age
+- Disability
+- Gender reassignment
+- Marital or civil partner status
+- Pregnancy or maternity
+- Race (including colour, nationality, ethnic or national origin)
+- Religion, religious belief or similar philosophical belief
+- Sex
+- Sexual orientation (ie homosexuality, bisexuality or heterosexuality)
+
+This list doesn't cover everything that might disadvantage someone. We're interested in fairness for everyone, not just meeting our legal obligations.
+
+## Definitions
+
+### Diversity
+
+This is a broader concept that builds upon the progress made through equal opportunities. Everyone is different and diversity means recognising, respecting and valuing the differences we each bring to work.
+
+Equal opportunities and diversity work together by identifying and addressing any inequalities and barriers faced by people and by valuing, learning and benefiting from the diverse cultures in society and our staff.
+
+### Direct discrimination
+
+This means treating someone less favourably than you would treat others because of a Protected Characteristic. For example, rejecting a job applicant because they are Jewish, or not promoting someone because they are gay. This includes any less favourable treatment because you perceive a person to have a Protected Characteristic (even though they do not in fact have it), or because they associate with a person or group who has a Protected Characteristic.
```

Comment:
Suggest we say `rejecting a job applicant because of their religion or religious beliefs` rather than calling out specific examples.